.circleci/config.yml

```diff
@@ -1041,8 +1045,8 @@ workflows:
                   filters:
                       branches:
                           only:
-                              - master
-                              - 6780-add-postman-tests-in-ci
+                              - master                              
+                              - cicd/run_postman_apim
```

Comment:
should not be overridden, otherwise it won't be usable with the target branch of the PR